### ISSUE TITLE
Evaluation or Pre-Trained AV Model?

### ISSUE THREAD

**Issue by romanos** [bill9800/speech_separation#3]
Hey @bill9800, thanks for sharing this repo. We've been working on some versions of this model since the AVSpeech dataset was published. Do you have any evaluation metrics, or do you mind sharing a pre-trained v2 model? We've been playing with a much narrower model trained on embeddings with 128 features, and it would be great to understand the differences a bit. 

Thanks again!

**Comment by bill9800** [comment]
Hi @romanos,
Due to the limitation of the resource, it's hard for me to train many epoch with large dataset. However, you can use my pretrained model to keep training. After some evaluation, I would say bigger data set is critical for the model.

Here are some pretrained model you can try. The learning rate =10e-6 and the gamma = 0.1 for objective function with 50K(6 epoch) and 230K(2 epoch) data set (construct by the around 500 videos). 
https://drive.google.com/drive/folders/1twAXGKGIn0RpqSpBPzC7eLztJaEU_sWp?usp=sharing

**Comment by romanos** [comment]
That's awesome 👍 , thanks so much!

I'll let you know how it goes, perhaps if we get better results we'll share our model back here as well.

**Comment by i-radwan** [comment]
Could any of you please share with us some of the test samples of the project ? I've tested the provided pertained model (230K(2 epoch)), but the two outputs tend to be exact to the original one (at least from what I can hear). 

Thank you
